__tests__/app.test.js

```diff
@@ -257,3 +257,53 @@ describe("GET /api/articles/:article_id/comments", () => {
     });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  const validComment = {
+    username: "butter_bridge",
+    body: "This is a test comment",
+  };
+
+  test("201: Should create a new comment with the provided username and body", async () => {
+    const response = await request(app)
+      .post("/api/articles/1/comments")
+      .send(validComment)
+      .expect(201);
+
+    expect(response.body).toEqual({
+      article_id: 1,
+      author: "butter_bridge",
+      body: "This is a test comment",
+    });
+  });
+
```

Comment:
Need another test of 201 when there're extra properties in the body but being ignored
e.g. `validComment = {
    username: "butter_bridge",
    body: "This is a test comment",
    extraProperty: "should be ignored"
  };`